appveyor.yml

```diff
@@ -48,16 +49,19 @@ test_script:
 - bundle exec rake acceptance:windows
 image:
   # Windows 2012 R2
-  - Visual Studio 2015
+  #- Visual Studio 2015
   # Windows 2016
   - Visual Studio 2017
 environment:
+  APPVEYOR_RDP_PASSWORD: Sensu1
   matrix:
     - PUPPET_GEM_VERSION: '~>5.x'
       PUPPET_REPO: puppet5
       RUBY_VERSION: 24-x64
-    - PUPPET_GEM_VERSION: '~>6.x'
-      PUPPET_REPO: puppet6
-      RUBY_VERSION: 25-x64
+#    - PUPPET_GEM_VERSION: '~>6.x'
```

Comment:
why are these commented out?

Reply:
See my recent comment yesterday, I am debugging appeveyor failures and it's easier to iterate over changes with fewer changes running. I had to pause my work last night due to issues with RDP in appveyor builds that allows me to log into VM and see why tests are failing. I will revert these changes once I get appveyor working.